app/components/forms/admin/settings/billing.js

```diff
@@ -12,17 +12,22 @@ export default class Billing extends Component.extend(FormMixin) {
       delay  : false,
       on     : 'blur',
       fields : {
-        adminBillingContactName: {
-          identifier : 'adminBillingContactName',
+        adminBillingEmail: {
```

Comment:
This was below phone rule. So please move it there. Try to make least amount of diff as possible to make it easy to review